src/data/schemas.js

```diff
@@ -103,6 +103,12 @@ export const METRIC_METADATA_SCHEMA = [
       "Defines the lifetime of the metric. Different lifetimes affect when the metrics value is reset.",
     helpLink: OPTIONAL_METRIC_PARAMS_DOCS,
   },
+  {
+    title: "First Seen",
+    id: "date_first_seen",
+    type: "value",
+    helpText: "The date when this metric was first seen.",
```

Comment:
Part of me feels like we should format the date a bit more, but I don't want to overcomplicate this PR. Let's put this in front of people and see what their feedback is.